player/command.c

```diff
@@ -7015,7 +7015,7 @@ void mp_option_change_callback(void *ctx, struct m_config_option *co, int flags,
         handle_force_window(mpctx, true);
         reinit_video_chain(mpctx);
         if (track)
-            reselect_demux_stream(mpctx, track, true);
+            queue_seek(mpctx, MPSEEK_RELATIVE, 0.0, MPSEEK_EXACT, MPSEEK_FLAG_NOFLUSH);
```

Comment:
Is there a reason you used the `MPSEEK_FLAG_NOFLUSH` flag here? I realized this is what causes the A/V desync in my testing. When it's 0, it's fine.

Reply:
No reason, honestly I think it's because I copied this from some other line that used this flag.
I'm not too familiar with this codebase...

Reply:
In that case, I think making it 0 is the better move here.